gallery/src/styles/NxFormLayout/NxFormHorizontalLayoutExample.tsx

```diff
@@ -37,6 +39,9 @@ export default function NxFormLayoutExample() {
 
   return (
     <form className="nx-form" onSubmit={onSubmit} aria-label="Horizontal Layout Example">
+      <NxTile.HeaderSubtitle>
```

Comment:
This is another case of a form in no specific type of container, same as https://github.com/sonatype/sonatype-react-shared-components/pull/735#discussion_r914795669

Reply:
Corrected here: [6a6aa83](https://github.com/sonatype/sonatype-react-shared-components/pull/735/commits/6a6aa8356c243bf8f7437c442254114bbc3209b0)